examples/imagenet/imagenet_benchmark.py

```diff
@@ -41,14 +32,12 @@ class ImagenetBenchmark(Benchmark):
   def test_8x_v100_half_precision(self):
     """Run ImageNet on 8x V100 GPUs in half precision for 2 epochs."""
     model_dir = tempfile.mkdtemp()
-    FLAGS.batch_size = 2048
-    FLAGS.half_precision = True
-    FLAGS.loss_scaling = 256.
-    FLAGS.num_epochs = 2
-    FLAGS.model_dir = model_dir
 
     start_time = time.time()
-    train.main([])
+    imagenet_lib.train_and_evaluate(model_dir=model_dir, batch_size=2048,
```

Comment:
I originally suggested to @AlexeyG to do the benchmarks this way but he persuaded me that it would be good to have the benchmarks exactly line up with the arguments you'd pass in the command line to the main train python file.

What do you think?

Reply:
Most of the cases, just set a few flag. 
Worst case, same as passing them to the example_lib.method().

I think I like calling the main idea better, Thanks for noticing this!